wp-content/themes/theme-starter-master/header.php

```diff
@@ -31,10 +31,12 @@
 <body <?php body_class(); ?>>
 
 		<header class="header" id="site-header" role="banner">
-    <nav id="main-nav" role="navigation">
-			<?php wp_nav_menu('menu=mainNav'); // create the mainNav menu inside Appearance menus and go to town -- for more on menus see: https://developer.wordpress.org/reference/functions/wp_nav_menu/ ?>
-		</nav>
-
+      <div class="main-navigation">
+        <a href="<?php echo home_url(); ?>"><img class= src="<?php echo get_template_directory_uri();?>/assets/img/Lakeview_logo.png" /></a>
```

Comment:
Not 100% sure how you'd get the right URL, but lookin at this PR I see:

![Uploading Screenshot 2019-04-14 18.27.58.png…]()

… so it looks like the image was uploaded through WP. Maybe there's a way to get the full path from a WP function? Or maybe the image needs to be added to WP in a different way or something?